doc/apple2enh.sgml

```diff
@@ -517,6 +517,14 @@ url="ca65.html" name="assembler manual">.
 </descrip><p>
 
 
+<sect1>Specifying file types for fopen<p>
+
+See section 
+<url url="apple2.html#ss9.4" name="Specifying file types for fopen">
+in the apple2 docoumentation.
```

Comment:
The `apple2enh` docs deliberately don't refer to the `apple2` docs but duplicate all content. Please adhere to this rule for new content.